source/precalculus/source/08-TE/05.ptx

```diff
@@ -12,6 +12,464 @@
     <subsection>
     <title>Activities</title>
 
+    <remark>
+      <p>
+      Trigonometry is the branch of mathematics that focuses on the relationships between the angles and sides of triangles. It has a wide range of applications across various fields, including science, engineering, architecture, and more. In this section, we will look at some common ways trigonometry is used.
+      </p>
+    </remark>
+
+    <activity xml:id="right-triangles-1">
+      <introduction>
+        <p>
+          A pilot signals to an air traffic controller that she wants to land. She wants to know what angle of fall to take when she is currently at <m>40,000</m> feet. Her plane is <m>100,000</m> feet from the runway, as the air traffic controller can see on the radar.
```

Comment:
```suggestion
          A pilot signals to an air traffic controller that she wants to land. She wants to know what angle of fall to take when she is currently at <m>40{,}000</m> feet. Her plane is <m>100{,}000</m> feet from the runway, as the air traffic controller can see on the radar.
```

This same change needs to be made elsewhere too, to prevent the extra space in the numbers.  See our [wiki](https://github.com/TeamBasedInquiryLearning/library/wiki/Style-Guide#math-typesetting)

Reply:
Also, I am not a pilot but I think "angle of descent" is more commonly used (and more descriptive) than "angle of fall"